### PR TITLE
[SDXL] Add SDXL image to image support

### PR DESCRIPTION
**Pull request by JingyaHuang**
- [x] Img2img pipeline
- [x] Inpaint pipeline
- [x] sdxl refiner export support (refiner unet)
- [x] Tests / Examples
- [x] Documentation update

### CODE REVIEW

File: docs/source/guides/models.mdx [hunk 1]
```diff
@@ -355,5 +364,112 @@ Now generate an image with a prompt on neuron:
   alt="sdxl generated image"
 />
 
+### Image-to-Image
+
+With `NeuronStableDiffusionXLImg2ImgPipeline`, you can pass an initial image, and a text prompt to condition gnerated images:
```
Comment (dacorvo) [inline docs/source/guides/models.mdx:369]:
```suggestion
With `NeuronStableDiffusionXLImg2ImgPipeline`, you can pass an initial image, and a text prompt to condition generated images:
```

File: docs/source/guides/models.mdx [hunk 2]
```diff
@@ -67,6 +67,12 @@ And the next time when you want to run inference, just load your compiled model
 As you see, there is no need to pass the neuron arguments used during the export as they are
 saved in a `config.json` file, and will be restored automatically by `NeuronModelForXXX` class.
 
+<Tip>
+
+When running inference for the first time, there is a warmup phase which would take 3x-4x latency than a regular run.
```
Comment (dacorvo) [inline docs/source/guides/models.mdx:72]: more latency ?

**Review by dacorvo — COMMENTED** [review]
Just one small nit, and a question: how do we identify the changes as compared to the original methods in the `pipeline_stable_diffusion_xx` files ?

**Comment by JingyaHuang** [comment]
@dacorvo for your question, do you mean the sd pipelines in `optimum` compared to those in `diffusers` or the difference between sd pipelines and sdxl pipelines?

**Comment by dacorvo** [comment]
> @dacorvo for your question, do you mean the sd pipelines in `optimum` compared to those in `diffusers` or the difference between sd pipelines and sdxl pipelines?

The difference between `optimum` and `diffusers`: I see that you give the link to the original files, but I was wondering if there were some hints in the code too to identify the lines you changed (I probably missed them).

**Review by dacorvo — APPROVED** [review]

**Comment by JingyaHuang** [comment]
@dacorvo there is for sd pipelines: https://github.com/huggingface/optimum-neuron/blob/main/optimum/neuron/pipelines/diffusers/pipeline_stable_diffusion_inpaint.py#L374

But I did not add it when working on sdxl...

**Pull request merged**